homeassistant/components/vizio/config_flow.py

```diff
@@ -336,26 +394,43 @@ def __init__(self) -> None:
             errors=errors,
         )
 
-    async def _pairing_complete(self, step_id: str) -> Dict[str, Any]:
-        """Handle config flow completion."""
+    async def async_step_pairing_complete_import(
+        self, user_input: Dict[str, Any] = None
+    ) -> Dict[str, Any]:
+        """Complete import config flow by displaying final message to show user access token and give further instructions."""
         if not self._must_show_form:
             return await self._create_entry_if_unique(self._data)
 
         self._must_show_form = False
         return self.async_show_form(
-            step_id=step_id,
+            step_id="pairing_complete_import",
             data_schema=vol.Schema({}),
             description_placeholders={"access_token": self._data[CONF_ACCESS_TOKEN]},
         )
 
-    async def async_step_pairing_complete(
+    async def async_step_tv_apps(
```

Comment:
Shouldn't this be part of the options flow? Our standard is to only include required config options in the main flow.

Reply:
That probably makes sense. I can submit a PR for that later today

Reply:
That probably makes sense. I can submit a PR for that later today